.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,155 @@
+        
+        Also, there is a running list of known issues in the [Podman Troubleshooting Guide](https://github.com/containers/podman/blob/main/troubleshooting.md), please reference that page before opening a new issue.
+        
+        If you are filing a bug against `podman build`, please instead file a bug against Buildah (https://github.com/containers/buildah/issues). Podman build executes Buildah to perform container builds, and as such the Buildah maintainers are best equipped to handle these bugs.
+
+        Commands you might need to run to create the issue
+          $ podman version
+          $ podman info
+          $ rpm -q podman
+  - type: dropdown
+    id: podman_version
+    attributes:
+      label: Podman Version
+      description: What version of our podman are you running?
+      options:
+        - main
+        - 4.3
+        - 4.2
+        - 4.1
+        - 4.0
+        - 3.4
+        - 3.3
+        - 3.2
+        - 3.1
+        - 3.0
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: full_podman_version
+    attributes:
+      label: Full Podman Version
+      description: Please enter the full version
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System 
+      description: What operating system are you using?
+      options:
+        - Fedora
+        - CentOS
+        - RHEL
+        - Ubuntu
+        - Arch
+        - MacOS
+        - Windows
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: os_version
+    attributes:
+      label: Operating System Version
+      description: Please enter your Operating System Version
+    validations:
+      required: true
+  - type: dropdown
+    id: arch 
+    attributes:
+      label: Architecture 
+      description: What architecture are you using to run podman?
+      options:
+        - x86_64 (Default)
+        - aarch64
+        - power
+        - s390x
+    validations:
+      required: true
+  - type: dropdown
+    id: podman_in_a_container 
+    attributes:
+      label: Podman in a container 
+      description: Please select Yes if you are running podman in a container
+      options:
+        - 'No'
+        - 'Yes'
+    validations:
+      required: true
+  - type: dropdown
+    id: privileged_rootless
+    attributes:
+      label: Privileged Or Rootless
+      description: Are you running the containers as privileged or non-root user?
+      options:
+        - Privileged
+        - Rootless
+  - type: dropdown
+    id: upstream_latest
+    attributes:
+      label: Upstream Latest Release
+      description: Have you tried running the [latest upstream release](https://github.com/containers/podman/releases/latest)
```

Comment:
NOTE, this link does not actually point to the latest release following semver. It chooses the latest release by date so it could link to outdated versions when we do backport releases.

Reply:
We can use https://github.blog/changelog/2022-10-21-explicitly-set-the-latest-release/ to mark the latest release, might require some work from @mheon based on how he is cutting the release.

Reply:
#16696 should solve this issue as well